### ISSUE TITLE
Accessibility (major): Landing page valid form field labels [prea-002]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/pre-assembly#1257]
## Description

`input id="batch_context_using_file_manifest"` should have a corresponding `<label for="batch_context_using_file_manifest">`

## Suggestion

Wrap the existing text in a `<label>`, like so:
```<label for="batch_context_using_file_manifest">I have a file manifest</label>```

https://docs.google.com/spreadsheets/d/105GozrKuWrnXaEk5FoP4UbzHSCTPgeEqPaPiHi1c950/edit#gid=0
